README-CN.md

```diff
@@ -121,6 +121,9 @@ English | [简体中文](./README-CN.md)
 
   * 验证是否成功通过访问 https://YOUR_GITHUB_USERNAME.github.io/microscope-v2/ i.e. if your github name is cryptape then replace YOUR_GITHUB_USERNAME with cryptape.
 
+# 使用docker 部署一个microscope
+  * docker build -t microscope . 构建docker镜像
```

Comment:
1. should use ordered list for flow steps 
2. should add a `#` before comment content